tests/test_order_SolidLiquid.py

```diff
@@ -35,21 +35,27 @@ def test_nlist(self):
 
         npt.assert_array_equal(nlist[:], comp.nlist[:])
 
-    def test_identical_environments(self):
+    comp_default = freud.order.SolidLiquid(6, q_threshold=0.7, solid_threshold=6)
+    comp_no_norm = freud.order.SolidLiquid(
+        6, q_threshold=0.3, solid_threshold=6, normalize_q=False
+    )
```

Comment:
Let's put these inside object construction calls inside the pytest.mark.parametrize block. I don't want random global variables floating around the class.